spectator/meter/age_gauge.go

```diff
@@ -0,0 +1,43 @@
+package meter
+
+import (
+	"fmt"
+	"github.com/Netflix/spectator-go/spectator/writer"
+)
+
+// AgeGauge represents a value that is the time in seconds since the epoch at which an event
+// has successfully occurred, or 0 to use the current time in epoch seconds. After an Age Gauge
+// has been set, it will continue reporting the number of seconds since the last time recorded,
+// for as long as the spectatord process runs. The purpose of this metric type is to enable users
+// to more easily implement the Time Since Last Success alerting pattern.
```

Comment:
There are no atlas-docs on this meter type, so I just inlined the spectatord description.